lib/rubocop/cop/mixin/configurable_max.rb

```diff
@@ -9,8 +9,10 @@ module ConfigurableMax
 
       def max=(value)
         cfg = config_to_allow_offenses
-        value = [cfg[max_parameter_name], value].max if cfg[max_parameter_name]
-        cfg[max_parameter_name] = value
+        current_max = cfg.dig(:exclude_limit, max_parameter_name)
```

Comment:
`dig` was introduced in Ruby 2.3. We still support down to Ruby 2.1 so you will have to use `cfg[:exclude_limit][max_parameter_name]`.